### PR TITLE
fix limit/offset

### PR DESCRIPTION
**Pull request by czpmango**
Fixes [https://jira.nebula-graph.io/browse/NG-368](https://jira.nebula-graph.io/browse/NG-368)

LimitSentence construction inside parser.yy is wrong.

### CODE REVIEW

**Review by laura-ding — APPROVED** [review]
Well done, could you fix it in v1.0 too?

**Review by Shylock-Hg — APPROVED** [review]

**Review by yixinglu — DISMISSED** [review]

**Review by Shylock-Hg — APPROVED** [review]

File: src/parser/parser.yy [hunk 1]
```diff
@@ -1770,6 +1770,9 @@ limit_sentence
         $$ = new LimitSentence($2, $4);
```
Comment (Shylock-Hg) [inline src/parser/parser.yy:1770]: How about this?

File: tests/tck/features/go/GO.feature [hunk 2]
```diff
@@ -1584,6 +1584,26 @@ Feature: Go Sentence
       | "Tony Parker" | "LaMarcus Aldridge" | "Manu Ginobili"     | "Tim Duncan"    |
       | "Tony Parker" | "LaMarcus Aldridge" | "LaMarcus Aldridge" | "Tim Duncan"    |
 
+  Scenario: Go and Limit
+    When executing query:
+      """
+      $a = GO FROM 'Tony Parker' OVER like YIELD like._src as src, like._dst as dst;
+      GO 2 STEPS FROM $a.src OVER like YIELD $a.src, $a.dst, like._src, like._dst OFFSET 1 LIMIT 2
```
Comment (Shylock-Hg) [inline tests/tck/features/go/GO.feature:1591]: pipe to limit

File: tests/tck/features/go/GO.feature [hunk 3]
```diff
@@ -2,6 +2,7 @@
 #
 # This source code is licensed under Apache 2.0 License,
 # attached with Common Clause Condition 1.0, found in the LICENSES directory.
+@czp
```
Comment (Shylock-Hg) [inline tests/tck/features/go/GO.feature:5]: Don't commit this.

File: tests/tck/features/go/GO.feature (hunk 2 above)
Comment (czpmango) [reply tests/tck/features/go/GO.feature]: Fixed, my mistake.

File: tests/tck/features/go/GO.feature (hunk 3 above)
Comment (czpmango) [reply tests/tck/features/go/GO.feature]: ok.

**Review by czpmango — COMMENTED** [review]
> Well done, could you fix it in v1.0 too?

ok.

File: tests/tck/features/go/GO.feature [hunk 4]
```diff
@@ -1584,6 +1584,24 @@ Feature: Go Sentence
       | "Tony Parker" | "LaMarcus Aldridge" | "Manu Ginobili"     | "Tim Duncan"    |
       | "Tony Parker" | "LaMarcus Aldridge" | "LaMarcus Aldridge" | "Tim Duncan"    |
 
+  Scenario: Go and Limit
+    When executing query:
+      """
+      $a = GO FROM 'Tony Parker' OVER like YIELD like._src as src, like._dst as dst;GO 2 STEPS FROM $a.src OVER like YIELD $a.src, $a.dst, like._src, like._dst | OFFSET 1 LIMIT 2
```
[the remark this answers is not in the document]
Comment (czpmango) [reply tests/tck/features/go/GO.feature]: Already modified.

**Review by laura-ding — APPROVED** [review]

**Review by Shylock-Hg — APPROVED** [review]

**Pull request merged**